src/login/register.jsx

```diff
@@ -104,6 +103,17 @@ const Register = () => {
               <label>Confirm Password</label>
             </div>
 
+            <div className="txtField">
+              <input
+                type="number"
+                autoComplete="off"
+                required
+                value={age}
+                onChange={(e) => setAge(e.target.value)}
+              />
+              <span></span>
```

Comment:
I suppose this span can go. Empty spans make for a lower quality HTML

Reply:
So the span is actually necessary. Its part of the text hovering effect if you take a look at the css